commit 9ea3ff987527b82492cc357fd0ba069484366087
Author: Keno Fischer <keno@juliacomputing.com>
Date:   Thu Nov 18 04:18:46 2021 -0500

    gvc.def: Don't override export name
    
    As reported in https://gitlab.com/graphviz/graphviz/-/issues/2156,
    the export name override breaks the win32/mingw32 autoconf build.
    By not specifying the export name, each toolchain should use its
    appropriate default, which should at least work, even if it's
    not consistent across toolchains.

diff --git a/lib/gvc/gvc.def b/lib/gvc/gvc.def
index 016eb5991..a91f4922d 100644
--- a/lib/gvc/gvc.def
+++ b/lib/gvc/gvc.def
@@ -1,5 +1,4 @@
-LIBRARY    "gvc"
-
+
 EXPORTS
 add_box    
 addPS    
